packages/components/src/components/button-toggle/button-toggle.scss

```diff
@@ -112,7 +112,9 @@
         content: '';
         height: 100%;
         top: 0;
+        /*rtl:begin:ignore*/
```

Comment:
this is to ignore certain scss changes in the post-rtlcss package. Some components may not need scss changes, by including this it will begin to ignore rtl package changes and we have to add `/*rtl:end:ignore*/` to end the ignore for rtl package changes.